src/.vuepress/config.js

```diff
@@ -48,6 +48,7 @@ export default {
         link: 'https://forum.chrultrabook.com/'
       }
     ],
+	headerDepth: 0,
```

Comment:
The spacing should match the rest of the file (4 spaces) and not be a tab lol

Reply:
Gotcha no problem, but the whole of the file is using 2 spaces rn?

Reply:
Sorry.. I just woke up and apparently can't count. 2 spaces then :)

Reply:
No worries, I'll use a more reasonable text editor in future:)